src/screens/profile/hooks.ts

```diff
@@ -0,0 +1,82 @@
+import {
```

Comment:
I believe we are not using this page anymore?

Reply:
Yes we no longer need these 4 files. Just keep it for now in case I need any information from them.